src/model/comment.ml

```diff
@@ -29,11 +29,20 @@ type inline_element =
   | `Reference of Reference.t * link_content
   | `Link of string * link_content ]
 
+type paragraph = inline_element with_location list
+
+type module_reference = {
+  module_reference : Reference.Module.t;
+  module_synopsis : paragraph option;
+}
```

Comment:
This module is used for two things: the output of the parser and the representation of comments inside Odoc. Here I intent to change the latter but it's a breaking change for users of the parser (eg. OCamlformat and Mdx).

A fix to that would be to duplicate the previous version of this module into the parser library and adding a conversion step.
We ought to extract the parser into its own package, that would be a breaking change too and would require that extra step anyway.

Reply:
Don't mdx and ocamlformat use `Ast.docs` rather than `Comment.docs`?

Reply:
You are right ! I invented the problem.